zeek/package.yaml

```diff
@@ -5,48 +5,1112 @@ author_icon: https://raw.githubusercontent.com/tenzir/library/main/author.svg
 package_icon: https://raw.githubusercontent.com/tenzir/library/main/zeek/package.svg
 categories:
 - sources
+- mappings
 
 description: |
   [Zeek](https://zeek.org) is an open-source network security monitor that
-  provides rich event logs.
+  provides rich logs about network activity.
 
-  This package makes it easy to onboard the logs that Zeek generates.
+  This package provides pipelines to onboard Zeek logs, as well as mapping
+  pipelines that translate Zeek logs to OCSF events.
 
 inputs:
-  listen-endpoint:
-    name: Listen Endpoint
-    description: A TCP endpoint where Zeek sends its logs.
+  url:
+    name: URL
+    description: The URL to the Zeek logs.
     type: string
-    default: 127.0.0.1:5555
-  log-file:
-    name: Log File Path
-    description: The path to the Zeek log to read.
-    type: string
-  log-format:
-    name: Log Format
+    default: file:///path/to/zeek/logs/**/*.log
```

Comment:
This is removing the ability to read Zeek from a TCP endpoint. Is that intentional?

Reply:
Yeah, it's not very common to do it this way.